components/nexus-core/src/main/java/org/sonatype/nexus/configuration/application/DefaultNexusConfiguration.java

```diff
@@ -246,8 +292,7 @@ public synchronized void loadConfiguration(boolean force)
       eventBus.post(loadEvent);
 
       if (loadEvent.isVetoed()) {
-        log.info(
-            vetoFormatter.format(new VetoFormatterRequest(loadEvent, log.isDebugEnabled())));
+        log.info(new DefaultVetoFormatter().format(new VetoFormatterRequest(loadEvent, log.isDebugEnabled())));
```

Comment:
Agree, though didn't want to incur too much change, just instead limit the visibility of these internal details that are likely to go away in the future.